libpromises/fncall.c

```diff
@@ -244,10 +248,9 @@ static FnCallResult CallFunction(EvalContext *ctx, const Policy *policy, const F
         rp = rp->next;
     }
 
-    char output[CF_BUFSIZE];
     if (argnum != RlistLen(expargs) && !(fncall_type->options & FNCALL_OPTION_VARARG))
     {
-        snprintf(output, CF_BUFSIZE, "Argument template mismatch handling function %s(", fp->name);
+        fprintf(stderr, "Argument template mismatch handling function %s(", fp->name);
```

Comment:
1. why `fprintf()` and not `Log()`?
2. is the ending closing parenthesis in the string a typo?